package/MDAnalysis/coordinates/timestep.pyx

```diff
@@ -493,20 +493,31 @@ cdef class Timestep:
         vectors lengths followed by their respective angle, or as three
         triclinic vectors.
 
-          >>> ts.dimensions
-          array([ 13.,  14.,  15.,  90.,  90.,  90.], dtype=float32)
-          >>> ts.triclinic_dimensions
-          array([[ 13.,   0.,   0.],
-                 [  0.,  14.,   0.],
-                 [  0.,   0.,  15.]], dtype=float32)
+          .. testsetup::
+
+          >>> import MDAnalysis as mda
+          >>> from MDAnalysis.tests.datafiles import TPR, XTC
+          >>> import numpy as np
+          >>> u = mda.Universe(TPR, XTC)
+          >>> ts = u.trajectory[0] 
+               
+          .. doctest::
+                    
+          >>> print(np.round(ts.dimensions))
+          [80. 80. 80. 60. 60. 90.]
+          >>> print(np.round(ts.triclinic_dimensions))
+          [[80.  0.  0.]
+           [ 0. 80.  0.]
+           [40. 40. 57.]]
 
         Setting the attribute also works::
-
+          
+          .. doctest::
+          
           >>> ts.triclinic_dimensions = [[15, 0, 0], [5, 15, 0], [5, 5, 15]]
-          >>> ts.dimensions
-          array([ 15.        ,  15.81138802,  16.58312416,  67.58049774,
-                  72.45159912,  71.56504822], dtype=float32)
-
+          >>> print(np.round(ts.dimensions))
```

Comment:
rounding here just gives the wrong output. Find a way to do this without `round`.

Reply:
@orbeckst I am getting this error

 ```
autodoc: failed to import module 'hole2' from module 'MDAnalysis.analysis'; the following exception was raised:
No module named 'mdahole2'
make: *** [Makefile:130: doctest] Error 2
```
I think mdahole2 uses python 3.11 or less I am using 3.12 so I am unable to install mdahole2. please check the error report I have attached in an earlier message. I will make the requested changes once this is resolved

Reply:
You might have to install in a Python 3.11 environment for testing. The development version is a work in progress and even though we strive to make it fully usable, this is not always possible until close to a release.